python/nav/web/syslogger/forms.py

```diff
@@ -48,40 +52,64 @@ def __init__(self, *args, **kwargs):
         self.fields['timestamp_from'].widget.format = DATEFORMAT[0]
         self.fields['timestamp_to'].widget.format = DATEFORMAT[0]
 
-        self.helper = FormHelper()
-        self.helper.form_tag = False
-        self.helper.layout = Layout(
-            Row(
-                Column(
-                    Fieldset(
-                        'Filter <a href="http://www.cisco.com/en/US/docs/ios/system/messages/guide/sm_cnovr.html"><i class="fa fa-info-circle"></i></a>',
-                        Row(
-                            Column(
-                                Field('facility', css_class='select2 medium-12'),
-                                css_class='medium-12',
-                            ),
-                            Column(
-                                Field('priority', css_class='select2'),
-                                css_class='medium-12',
-                            ),
-                            Column(
-                                Field('mnemonic', css_class='select2'),
-                                css_class='medium-12',
-                            ),
-                            Column(
-                                Field('origin', css_class='select2'),
-                                css_class='medium-12',
-                            ),
-                            Column(
-                                Field('category', css_class='select2'),
-                                css_class='medium-12',
-                            ),
-                            Column('timestamp_from', css_class='medium-12'),
-                            Column('timestamp_to', css_class='medium-12'),
-                            Column('show_log', css_class='medium-12'),
-                        ),
-                    ),
-                    css_class='medium-12',
-                ),
-            ),
+        self.fields['facility'].widget.attrs.update({"class": "select2 medium-12"})
```

Comment:
I think the `medium-12` is mistake from earlier, because removing it does not make a difference on how it looks